io-engine/src/grpc/v0/mayastor_grpc.rs

```diff
@@ -164,13 +164,24 @@ impl From<LvsError> for Status {
             LvsError::ReplicaShareProtocol {
                 ..
             } => Status::invalid_argument(e.to_string()),
-
             LvsError::Destroy {
                 source, ..
             } => source.into(),
             LvsError::Invalid {
                 ..
             } => Status::invalid_argument(e.to_string()),
+            LvsError::PoolNotFound {
+                ..
+            } => Status::not_found(e.to_string()),
+            LvsError::PoolCreate {
+                source, ..
+            } => if source == Errno::EEXIST{
```

Comment:
nit: should have space:
```suggestion
            } => if source == Errno::EEXIST {
```
and same for EINVAL below.

Reply:
Incorporated